### DISCUSSION TITLE
针对个人如何选云服务商和服务器 存储

### DISCUSSION THREAD

**Discussion by wanghaisheng** [cloudreve/cloudreve#1715] [General]
请教一下，想弄一个取代百度网盘 和阿里云盘，
一方面是放一些不怎么会看的视频 大概100g左右
一方面是放一些pdf文件 大概30g左右 走学术文献管理的webdav链接

有没有性价比比较高的方案推荐

**Comment by weiyongzenqi** [comment]
分享下我的,之前9.9刀买的一年某云服务商黑五活动 1c1g 40G硬盘 100Mbps带宽 无限流量 配合E5 5TB的OneDrive 再有一台9.9刀/年的机器专门用于反向代理Onedrive基本可以实现24小时在线放放视频没问题 容量现在用了300多GB了 存的某些内容没做加密/压缩也没删我的,正常访问

**Reply by wanghaisheng** [reply]
我没有啥视频 主要是几十G的pdf  E5 5TB的onedrive咋弄的
你这套搭建有教程吗

**Reply by weiyongzenqi** [reply]
E5是微软的E5开发者账户，你百度搜就能找到相关教程，不过这玩意儿需要续订的，上次清明节把一大批人账号都封了，续订是免费的，不过要让微软认为你在进行相关开发，具体怎么做你可以自己搜搜

**Reply by wanghaisheng** [reply]
账号搞上了  我去看看饭袋 是不是直接有那种跑在cf的 谢谢分享

**Reply by weiyongzenqi** [reply]
> 账号搞上了 我去看看饭袋 是不是直接有那种跑在cf的 谢谢分享

不用谢,反代的话就nginx,反代你E5 od下载的时候的域名(不过要注意把反代缓存关掉 
配置文件里加proxy_buffering off;
proxy_cache off;
proxy_set_header X-Forwarded-Proto $scheme; 不然的话用cloudreve在线播放流媒体会有很大问题,如动一下进度条直接全部重新加载,缓存关掉就什么问题都没有了,浏览器安装为pwa应用用起来比国内云盘客户端香很多),cf也可以,不过上次测试了下速度不行,懒着做ip优选了,就用的别的服务器

**Reply by wanghaisheng** [reply]
9.9刀/年.机器你哪个平台的 我去看看

**Reply by weiyongzenqi** [reply]
> 9.9刀/年.机器你哪个平台的 我去看看

不是广告 我是赶上去年raksmart黑五的活动 9.9刀1c1g40G盘100Mbps带宽无限流量 现在已经没了,cloudcone有大部分时间都在10.99刀1c0.5G20G1000Mbps 2TB流量的机器(以前是9.9,今年涨价了),racknerd也有差不多价格差不多的套餐的机器,我只是说我知道的用过的,不保证任何东西哈 如果你纯粹是自己用 1c0.5g20G盘的机器开2G的swap够用了,不过用离线下载功能的话也会限制在剩余空间上,不然存不下,反代的话可以另开个机子做,也可以用cf(速度看情况)

**Reply by weiyongzenqi** [reply]
> 9.9刀/年.机器你哪个平台的 我去看看

E5记得搞续订,不然号没了30天内需要数据迁移

**Reply by wanghaisheng** [reply]
我准备找个powerapp的代码 在365上部署一下 应该就不会收回了
那个机器100块以内无所谓 我正好可以部署一下wp练练手
